cmd/kubeadm/app/cmd/phases/uploadconfig.go

```diff
@@ -19,70 +19,127 @@ package phases
 import (
 	"fmt"
 
+	"github.com/golang/glog"
 	"github.com/pkg/errors"
-	"github.com/spf13/cobra"
-
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	kubeadmapiv1beta1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1"
+	clientset "k8s.io/client-go/kubernetes"
+	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow"
 	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	kubeletphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/kubelet"
+	patchnodephase "k8s.io/kubernetes/cmd/kubeadm/app/phases/patchnode"
 	"k8s.io/kubernetes/cmd/kubeadm/app/phases/uploadconfig"
-	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
-	configutil "k8s.io/kubernetes/cmd/kubeadm/app/util/config"
-	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
 	"k8s.io/kubernetes/pkg/util/normalizer"
 )
 
 var (
-	uploadConfigLongDesc = fmt.Sprintf(normalizer.LongDesc(`
-		Uploads the kubeadm init configuration of your cluster to a ConfigMap called %s in the %s namespace. 
+	uploadKubeadmConfigLongDesc = fmt.Sprintf(normalizer.LongDesc(`
+		Uploads the kubeadm ClusterConfiguration to a ConfigMap called %s in the %s namespace. 
 		This enables correct configuration of system components and a seamless user experience when upgrading.
 
 		Alternatively, you can use kubeadm config.
-		`+cmdutil.AlphaDisclaimer), kubeadmconstants.KubeadmConfigConfigMap, metav1.NamespaceSystem)
+		`), kubeadmconstants.KubeadmConfigConfigMap, metav1.NamespaceSystem)
 
-	uploadConfigExample = normalizer.Examples(`
+	uploadKubeadmConfigExample = normalizer.Examples(`
 		# uploads the configuration of your cluster
 		kubeadm alpha phase upload-config --config=myConfig.yaml
```

Comment:
I think that change `alpha` to `init`

Reply:
https://github.com/kubernetes/kubernetes/pull/71053